test/smoke/src/areas/positron/dataexplorer/dataexplorer.test.ts

```diff
@@ -17,9 +17,8 @@ describe('Data Explorer #web #win', () => {
 	describe('Python Pandas Data Explorer #pr', () => {
 
 		before(async function () {
-
+			this.timeout(120000);
```

Comment:
Is this supposed to be on the 100x100 test?  curious why these were added here.

Reply:
Because I saw the test exceed past the global timeout of 1 min a couple times. Maybe I could have made it 90s though. I was feeling aggressive this morning.

Reply:
oh, ok.  I thought 100x100 was the slow one.

Reply:
i think there are more than a couple slow ones...